bgpd/bgp_ecommunity.c

```diff
@@ -667,6 +670,11 @@ ecommunity_ecom2str (struct ecommunity *ecom, int format)
           break;
 
         case ECOMMUNITY_ENCODE_OPAQUE:
+          if(filter == ECOMMUNITY_ROUTE_TARGET)
```

Comment:
API extension.
This function is used for doing the show running.
And I use this function for dumping extra parameters, that are contained in extended communities, but not set with the standard rt import/export vty.